tests/nbrun.py

```diff
@@ -0,0 +1,35 @@
+import sys
+import os
+from pathlib import Path
+import nbconvert
+import nbformat
+from nbconvert.preprocessors import ExecutePreprocessor
+import glob
+
+
+def run_notebook(in_filepath):
+    if not in_filepath.is_file():
+        raise IOError('File "%s" not found.' % in_filepath)
+    in_filepath = in_filepath.resolve()
+
+    run_path = str(in_filepath.parent)
+
+    nb = nbformat.read(in_filepath.open(), as_version=4)
+    ep = ExecutePreprocessor(
+        timeout=3600, kernel_name='python%d' % sys.version_info[0])
```

Comment:
The timeout here is in seconds?  If so then I suggest that we reduce this a bit to something like 300 (five minutes)

Reply:
Yes, it is in seconds. I meant to change it but forgot. I agree with 300.